serverless-scheduler-proxy/main.go

```diff
@@ -117,6 +118,23 @@ func rewriteBotContainerURL(c botConfig) func(*http.Request) {
 	}
 }
 
+func rewriteBotCanaryURL(c botConfig) func(*http.Request) {
```

Comment:
I had to stare at this code for longer than I'd care to admit before I realized I wasn't looking at JavaScript.